microservice/src/main/java/nearsoft/skytouch/microservice/subscriber/Subscriber.java

```diff
@@ -25,15 +25,26 @@
     @RabbitListener(queues = "requestProductsQueue")
     public String subscribeToRequestQueue(@Payload String requestMessage, Message message) {
         LOGGER.debug("Request received");
-        List<Product> products = this.productService.getProducts();
-        return productJSONSerializer.serializeList(products);
+        List<Product> products = null;
+        try {
+            products = this.productService.getProducts();
+            return productJSONSerializer.serializeList(products);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Don't use printStackTraces... use logger here

Reply:
Ok thanks i going to change it :D